Code/GraphMol/ChemReactions/ReactionRunner.cpp

```diff
@@ -521,6 +562,38 @@ void addReactantNeighborsToProduct(
             const Bond *origB =
                 reactant.getBondBetweenAtoms(lreactIdx, *nbrIdx);
             addMissingProductBonds(*origB, product, mapping);
+          } else {
+            // both mapped atoms are in the match.
```

Comment:
You're right that it should be in the docs. I will try and get that done before the release.

That last example falls into the bucket of "not bound in reactant template"->"not bound in the product template". So that won't have any impact on any C:1->C:2 bond that may be in the reactant itself; I think